src/main/java/datamodels/AccountIdInfo.java

```diff
@@ -0,0 +1,7 @@
+package datamodels;
+
+public class AccountIdInfo extends IdInfo{
```

Comment:
abstract class does not need constructor

Reply:
I think we're unclear on what the purpose of this class is supposed to be. Also if it is meant to be abstract then declare it with the abstract keyword.

Reply:
my apologies, an abstract class should have a constructor in order to initialize fields and call super().